Applet/src/com/android/javacard/keymaster/KMKeymasterApplet.java

```diff
@@ -3257,6 +3258,12 @@ private void processImportKeyCmd(APDU apdu) {
 
     data[CERTIFICATE] = KMArray.instance((short)0); //by default the cert is empty.
     data[ORIGIN] = KMType.IMPORTED;
+    //ID_IMEI should be present if ID_SECOND_IMEI is present
+    short attIdTag = KMKeyParameters.findTag(KMType.BYTES_TAG, KMType.ATTESTATION_ID_SECOND_IMEI, data[KEY_PARAMETERS]);
+    if (attIdTag != KMType.INVALID_VALUE) {
+        KMTag.assertPresence(data[KEY_PARAMETERS], KMType.BYTES_TAG, KMType.ATTESTATION_ID_IMEI,
```

Comment:
Rethinking about placing this in addAttestationIds as it would be a common place and no need of duplication.